pkg/controller/launcher/launcher_controller.go

```diff
@@ -225,8 +225,13 @@ func (r *ReconcileLauncher) Reconcile(request reconcile.Request) (reconcile.Resu
 			data["launcher.keycloak.url"] = instance.Spec.OAuth.KeycloakURL
 			data["launcher.keycloak.realm"] = instance.Spec.OAuth.KeycloakRealm
 			data["launcher.keycloak.client.id"] = instance.Spec.OAuth.KeycloakClientID
+			data["launcher.missioncontrol.openshift.password"] = ""
+			data["launcher.missioncontrol.openshift.username"] = ""
 		} else {
 			data["launcher.oauth.openshift.url"] = instance.Spec.OpenShift.ConsoleURL + "/oauth/authorize"
+			data["launcher.keycloak.url"] = ""
+			data["launcher.keycloak.realm"] = ""
+			data["launcher.keycloak.client.id"] = ""
```

Comment:
Shouldn't `launcher.missioncontrol.openshift.password` and `launcher.missioncontrol.openshift.username` be empty here as well?

Reply:
I don't know you said it should be empty when keycloak was used, this is when it's using oauth from openshift

Reply:
If OAuth from Openshift is used this should be also empty. I think the only condition that would have those values set is when neither OAuth nor Keycloak are set